### ISSUE TITLE
CSS drowdown do not open

### ISSUE THREAD

**Issue by RaphArbuz** [gmetais/YellowLabTools#92]
On this page for instance: http://yellowlab.tools/result/e5kvtzggoc/rule/cssComplexSelectors the dropdown menus referring to CSS complex selectors do not open (with Chrome 43.0.2357.134)

**Comment by gmetais** [comment]
Hi @RaphArbuz,

Thank you! I found the reason for this problem and fixed it.

**The reason:** 
Last deploy (yesterday) failed silently to install some NPM dependencies. Here is the result of `npm list`:

```
npm ERR! missing: imagemin@3.2.0, required by yellowlabtools@1.7.3
npm ERR! missing: phantomas@1.11.0, required by yellowlabtools@1.7.3
npm ERR! not ok code 0
```

Phantomas 1.11 was not there and YLT was then using the global Phantomas instead, which on the public instance server was a 1.6 version, quite old. 

**Help needed:**
I need to find a way to make sure my deploy shell script fails if `npm install` fails on one or more modules. Here is the current deploy script, if someone knows how to do that: https://github.com/gmetais/YellowLabTools/blob/master/server_config/server_update.sh#L15

I'm keeping the issue open until this is fixed.

**Comment by gmetais** [comment]
Fixed by #93.
